content/rs/security/certificates/ocsp-stapling.md

```diff
@@ -0,0 +1,96 @@
+weight: 50
+alwaysopen: false
+categories: ["RS"]
+aliases: ["/rs/security/ocsp-stapling"]
+---
+
+OCSP ([Online Certificate Status Protocol](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol)) lets a client or server verify the status (valid, revoked, or unknown) of a certificate maintained by a third-party [certificate authority (CA)](https://en.wikipedia.org/wiki/Certificate_authority).
+
+When a client or server needs to check whether a certificate is still valid or has been revoked, it can send a request to the CA's OCSP server (also called an OCSP responder). The OCSP responder checks the certificate's status in the CA's [certificate revocation list](https://en.wikipedia.org/wiki/Certificate_revocation_list) and sends the status back as a signed and timestamped response.
+
+## OCSP stapling overview
+
+ With OCSP enabled, the Redis Enterprise server regularly polls the CA's OCSP responder for the certificate's status. After it receives the response, the server caches this status until its next polling attempt.
+
+ When a client tries to connect to the Redis Enterprise server, they perform a [TLS handshake](https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_handshake) to authenticate the server and create a secure, encrypted connection. During the TLS handshake, [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling) lets the Redis Enterprise server send (or "staple") the cached certificate status to the client.
+
+If the stapled OCSP response confirms the certificate is still valid, the TLS handshake succeeds and the client connects to the server.
+
+The TLS handshake fails and the client blocks the connection to the server if the stapled OCSP response indicates either:
+
+- The certificate has been revoked.
+
+- The certificate's status is unknown. This can happen if the OCSP responder fails to send a response.
+
+## Set up OCSP stapling
+
+Configure and enable OCSP stapling for your Redis Enterprise cluster with the [admin console](#admin-console-method), the [REST API](#rest-api-method), or [`rladmin`](#rladmin-method).
+
+While OCSP is enabled, the server always staples the cached OCSP status when a client tries to connect. It is the client's responsibility to use the stapled OCSP status. Some Redis clients, such as [Jedis](https://github.com/redis/jedis) and [redis-py](https://github.com/redis/redis-py), already support OCSP stapling, but others might require additional configuration.
+
+### Admin console method
+
+To set up OCSP stapling with the Redis Enterprise admin console:
+
+1. Use [`rladmin`]({{<relref "/rs/references/rladmin#cluster-certificate">}}) or the [REST API]({{<relref "/rs/references/rest-api/requests/cluster/update-cert">}}) to [replace the proxy certificate]({{<relref "/rs/security/certificates/updating-certificates">}}) with a certificate signed by your third-party CA.
+
+1. Sign in to the Redis Enterprise admin console.
+
+1. Go to **settings > OCSP**.
```

Comment:
Could not find this option in Settings

Reply:
OCSP is only in the Blanc release. You might have an older version of RS.